openmmtools/mcmc.py

```diff
@@ -469,19 +465,19 @@ def apply(self, thermodynamic_state, sampler_state):
         move.apply(thermodynamic_state, sampler_state)
 
     def __getstate__(self):
-        serialization = dict()
-        for i, (move, weight) in enumerate(self.move_set):
-            serialization['move' + str(i)] = utils.serialize(move)
-            serialization['move' + str(i) + '_weight'] = weight
-        return serialization
+        serialized_moves = [utils.serialize(move) for move, _ in self.move_set]
+        weights = [weight for _, weight in self.move_set]
+        return dict(moves=serialized_moves, weights=weights)
```

Comment:
This uses a different key to store the move list than in the previous case. If these are meant to be used together, or at least passed into each other, then I think you will want to change `moves` to `move_list`